compiler/circlechef/circle/src/CircleOpRegistry.h

```diff
@@ -58,6 +58,7 @@ class CircleOpRegistry
     REG_TFL_OP(BATCH_MATMUL, CircleOpBatchMatMul);
     REG_TFL_OP(BCQ_FULLY_CONNECTED, CircleOpBCQFullyConnected);
     REG_TFL_OP(BCQ_GATHER, CircleOpBCQGather);
+    REG_TFL_OP(CIR_GRU, CircleOpCircleGRU);
```

Comment:
```suggestion
    REG_TFL_OP(GRU, CircleOpGRU);
```

Reply:
or,
1/ to prepare tflite introduces GRU,
2/ two `Circle` looks odd
```suggestion
    REG_TFL_OP(CIR_GRU, CircleOpCirGRU);
```

Reply:
I think second variant better to avoid future problems with tflite GRU operation

Reply:
Done